lib/bridge/instance.js

```diff
@@ -107,11 +116,14 @@ IrcBridge.prototype.run = Promise.coroutine(function*(port) {
             This release will use values from this config file. This will produce
             a fatal error in a later release.`
         );
+        this.domain = this.config.appService.homeserver.domain;
+        this.appServiceUserId = (
+            "@" + (this.config.appService.localpart || IrcBridge.DEFAULT_LOCALPART) + ":" +
+            this.config.homeserver.domain
```

Comment:
Use this.domain?

Right now we have three things:

this.domain == this.config.appService.homeserver.domain ?= this.config.homeserver.domain